examples/stable-diffusion/train_text_to_image_sdxl.py

```diff
@@ -1219,17 +1220,16 @@ def compute_time_ids(original_size, crops_coords_top_left):
                         accelerator.save_state(save_path)
                         logger.info(f"Saved state to {save_path}")
 
-            if global_step % args.logging_step == 0:
-                train_loss_scalar = train_loss.item()
-                accelerator.log({"train_loss": train_loss_scalar}, step=global_step)
+                if (global_step - 1) % args.logging_step == 0 or global_step == args.max_train_steps:
```

Comment:
From the original https://github.com/huggingface/diffusers/blob/v0.25.1/examples/text_to_image/train_text_to_image_sdxl.py#L1141, logging is outside of accelerator.sync_gradients. Do wee need to put inside of sync_gradient?